faqs/galaxy/tools_tool_doesn't_recognize_any_input_datasets.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Tool doesn't recognize any input datasets
+area: tools
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+This is usually a simple datatype assignment incompatibility between the dataset and the tool.
+Expected input datatype format is explained on the Tool form itself on the "Please provide a value for this option" section or on the information section below the Execute button.
+
+1. Click on the pencil icon {% icon galaxy-pencil %} found on the dataset to reach the **Edit Dataset Attributes** form.
```

Comment:
Theses 4 steps are basically the same as the existing ["Changing datatype" FAQ](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_change_datatype.html)

The other common reason the tool won't suggest the desired input, is that they want to input a collection, but have not clicked on the button for that (https://training.galaxyproject.org/training-material/faqs/galaxy/tools_select_collection.html)

Maybe we can rewrite this FAQ to point to (or include directly) the existing datatype changing FAQ to avoid duplication, and same for the collection input